tests/fixtures/sample_lib/modules/sample_task/sample_implementation.py

```diff
@@ -50,6 +52,8 @@ def run(
         sample_input: SampleInputType,
         throw: bool = False,
         error: Optional[str] = None,
+        request_id: Optional[str] = None,
```

Comment:
NIT: This is a test, so it's fine, but we should probably add these (and the others) to the docstring